src/nvim/generators/gen_unicode_tables.lua

```diff
@@ -208,9 +208,7 @@ local build_width_table = function(ut_fp, dataprops, widthprops, widths,
       -- But use all chars from a range.
       local dp = dataprops[dataidx]
       if (n_last > n) or (not (({Mn=true, Mc=true, Me=true})[dp[3]])) then
-        if start >= 0 and end_ + 1 == n then
-          -- Continue with the same range.
-        else
+        if not (start >= 0 and end_ + 1 == n) then
```

Comment:
same here, the old code was intentional documentation. Also when adding `-- luacheck: ignore` it should only ignore the specific luacheck code number. In this case it's 542, see http://luacheck.readthedocs.io/en/stable/warnings.html

```lua
if start >= 0 and end_ + 1 == n then  -- luacheck: ignore 542
  -- Continue with the same range.
else
```